readme.md
////////////////////////////////////////////////////////////////
//			INSTALL GUIDE - DAYZ ENDGAME
////////////////////////////////////////////////////////////////

YOU MUST HAVE 

STEAM BETA 125548

BEFORE USING THIS!!

RUN THE FULLDB.SQL ON YOUR DB 



////////////////////////////////////////////////////////////////
//				ADMIN SETUP
////////////////////////////////////////////////////////////////
ADMIN TOOLS MENU

1. open "admintools\AdminList.sqf"
2. change 120958982 or (######) to your player ID

////////////////////////////////////////////////////////////////
//				KNOWN BUGS
////////////////////////////////////////////////////////////////
1. there is some delay when going from one trader to the next trader.
2. Sometimes when on fresh server start player gets BE kick for setpos just rejoin.
3. Ground fog / Wind Dust can cause low FPS and server lag
4. there are a few RPT errors that may spam your RPT


////////////////////////////////////////////////////////////////
//				TURN SCRIPTS ON/OFF
////////////////////////////////////////////////////////////////
1. Open scriptcontrol.sqf and edit to your liking
true = on
false = off
	
////////////////////////////////////////////////////////////////
//			CHANGE WELCOME CREDITS	
////////////////////////////////////////////////////////////////
1.	open "scripts\ServerWelcomeCredits.sqf"
2.	read the comments in the file.

////////////////////////////////////////////////////////////////
//			CHANGE WATERMARK	
////////////////////////////////////////////////////////////////
1.	open "init.sqf"
2.	find server_name = "Sheeps Epoch Repack";
3.	change Sheeps Epoch Repack to your liking


////////////////////////////////////////////////////////////////
//			CHANGE LOAD SCREEN	
////////////////////////////////////////////////////////////////
1.	To change the loading screen make a JPG image 
	make sure it is one of following resolutions 512x256, 1024x512, 2048x1024

2.	save it in pictures in the mission folder as "loadscreen.jpg"
	(overwrite the one thats there, its the repack default)

////////////////////////////////////////////////////////////////
//			CHANGE DEATH SCREEN	
////////////////////////////////////////////////////////////////
1.	To change the death screen make a PAA image 
	(first make a JPG then open it with TexView2 and saves as PAA)
	make sure it is one of following resolutions 512x256, 1024x512, 2048x1024

2.	save it in pictures in the mission folder as "deathscr.paa"
	(overwrite the one thats there, its the repack default)	
	
////////////////////////////////////////////////////////////////
//			CHANGE DEBUG MONITOR
////////////////////////////////////////////////////////////////
1.	open "scripts\DebugMonitor\debug_monitor.sqf"

////////////////////////////////////////////////////////////////
//			CHANGE DEBUG MONITOR SERVER RESTART TIME
////////////////////////////////////////////////////////////////
1.	open "scripts\DebugMonitor\debug_monitor.sqf"
2.  find this line (round(240-(serverTime)/60)) and change the 240 to how many minutes before restart
	EG: if you restart every 2 hours you would change 240 to 120
	
////////////////////////////////////////////////////////////////
//			BATTLE EYE
////////////////////////////////////////////////////////////////
First relesae to support battle eye!
Massive thanks to Kaysio (aka J0k3r)!
1.	Replace you current BE filters with the ones included.

////////////////////////////////////////////////////////////////
//			AI CUSTOMISING
////////////////////////////////////////////////////////////////
The repack uses DZAI and WAI see the DZAI or WAI forum for help adjusting your AI!

////////////////////////////////////////////////////////////////
//			MISSION CUSTOMISING
////////////////////////////////////////////////////////////////
The repack uses DZMS and WAI see the DZMS or WAI forum for help adjusting your missions!

////////////////////////////////////////////////////////////////
//			ESS DONOR CLASS
////////////////////////////////////////////////////////////////
You need to UNPBO the server and edit dayz_server\init\server_functions.sqf

donorClassLvl1 = ["0","0","0"];
donorClassLvl2 = ["0","0","0"];
donorClassLvl3 = ["120958982","110704838","0"];

change 0 to the donors UID to give them access


////////////////////////////////////////////////////////////////
//			ESS DONOR LOADOUT
////////////////////////////////////////////////////////////////
You need to UNPBO the server and edit dayz_server\init\server_functions.sqf

donorListClass = [
	"120958982",
	"110704838",
	"108784838"
];

change 120958982 to the donors UID to give them access

donorListClasses = [
	["ebay's Loadout","GUE_Soldier_2_DZ",["100Rnd_762x51_M240","100Rnd_762x51_M240","30Rnd_9x19_UZI_SD","30Rnd_9x19_UZI_SD","30Rnd_9x19_UZI_SD","ItemAntibiotic","ItemBloodbag","ItemEpinephrine","ItemMorphine","Skin_GUE_Soldier_Sniper_DZ","ItemBandage"],["Mk_48_DZ","UZI_SD_EP1","Binocular_Vector","NVGoggles","ItemCompass","ItemHatchet","ItemKnife","Itemmatchbox","Itemetool","ItemSledge","ItemWatch","ItemGPS","ItemCrowbar"],"DZ_Backpack_EP1",["5Rnd_86x70_L115A1","5Rnd_86x70_L115A1","5Rnd_86x70_L115A1"],["BAF_LRR_scoped_W"],0,0],
	[],
	[]
];

adjust the loadout and name to your donors request or your donor system


////////////////////////////////////////////////////////////////
//			LOOT CUSTOMISING
////////////////////////////////////////////////////////////////
To change loot open the following 3 files

scripts\customloot\CfgBuildingLoot.hpp
scripts\customloot\cfgLoot.hpp
scripts\customloot\CfgLootSmall.hpp

EXAMPLE 
Open extras\custom_loot\Configs\cfgLoot.hpp
At the top of the file you will see this
trash[] = {
		{"TrashTinCan",0.5}
		,{"TrashJackDaniels",0.05}
		,{"ItemSodaEmpty",0.25}
		,{"ItemTrashToiletpaper",0.1}
		,{"ItemTrashRazor",0.1}
	};

So whats it mean?
Basically
“TrashTinCan” has a 0.5 chance to spawn in the loot table trash
So if you wanted it to appear more often in trash piles increase the number

To add a new item to spawn add it like so
trash[] = {
		{"TrashTinCan",0.5}
		,{"TrashJackDaniels",0.05}
		,{"ItemSodaEmpty",0.25}
		,{"ItemTrashToiletpaper",0.1}
		,{"ItemTrashRazor",0.1}
		,{"ItemTrashRazor",0.1} <---- New Item
	};
	
////////////////////////////////////////////////////////////////
//				HELP / RULES
////////////////////////////////////////////////////////////////
	to change the rules / help 
	1. open scripts\help/help.html and edit it 
	1. open scripts\help/help_txt.html and edit it 
	
	(you can use standard HTML to fancy it up :P )
	
////////////////////////////////////////////////////////////////
//				DEBUG / RPT COLOR CODE
////////////////////////////////////////////////////////////////
import RPTcolor.xml into notepad++ to color you rpt logs (makes it easier to find errors and admin abuse etc

Errors are red
Admin Tools are Pink

all other colors are normal loading stuff just split up to make it easier to see

////////////////////////////////////////////////////////////////
//			CREDITS
////////////////////////////////////////////////////////////////
	Kaysio (aka J0k3r)!
	SchwEde (aka shinysonic)
	Sgt M Everlast
	MattL
	AlexLawson
	Carl
	The entire Sheeps Epoch Repack team
	Each addon/script creator
	The entire EPOCH team
	The entire ARMA2 team
	Dean "Rocket" Hall & the entire DayZ team
	The entire Open Dayz team & comunnity!
	
	
////////////////////////////////////////////////////////////////
//			SCRIPTS/ADDONS IN REPACK
////////////////////////////////////////////////////////////////
Advanced Alchemy Building
Admin Tools
Debug Monitor 
Take clothes 
Tent healing
Zombie Bait
Zombie Bomb
Suicide
DZAI (dayz AI)
DZMS (dayz mission system)
Burn Tents
Drink Water
Fast Roping
Building Snapping
Heli Lift
Towing
CAGN Safezones
Animated Heli Crashes
Auto Refuel
Bury Body
Regen Blood
Snow
Ground Fog
Anti-Zombie Emitter
Street Lights
House Lights
Tower Lights
Animated AN2 crashes
AN2 carepackage drops
Welcome Credits
Custom loading Screen
Custom Loot
3 new DZMS Missions (construction crate, Supply crate, treasure crate {AI Guarded} )
ESS spawnselect
EVR Blowout (from namalsk need ItemNVG be protected)
Mv22 Folding wings
Animated Armoured SUV Turrent
Gems at traders (General Store at each main trader)
Arrest Player 
(disabled in safe zones must have 5000 humanity or -5000 humanity or be admin and scrap metal)
Hero perks
Harvest Hemp (buy/sell at black markets - right click knife near hemp field)
Ixxos Awesome custom buildings
Admin Tool  RPT Tracking ( EG: [ADMIN TOOLS] - SPAWN cinder_wall_kit - Admin Name: DUDE UID: 111111 Position: 1200.0, 400.4") 
WAI (Wicked AI 0.73)
Deploy Bike + Upgrade to Motorbike then Mozzie (right click toolbox)
Added radio to rare loot (military loot and police zombie loot, very low chance) and black market trader for 100oz briefcase
Custom Death Screen
DZGM (dayz group management)
VASP (Vehicle and skin preview)
Bank Robbery
Walk Amongst Dead
Lots more custom buildings
call carepackage (right click 100oz briefcase)
call carepackage to map location (right click 100oz briefcase)
Journal
Elevator
Adjustable fog/view distance
Fast Trading
Trade from Vehicles
Trade from backpacks
Vector build
Admin build
JAEM ( just another evac mod)

Changelog.txt
0.0.0.2
CHANGED - lowered all loot chances to 0.01 imprpe for balancing
ADDED - Miltary Vehicles to traders and vehicle spawns
ADDED - origins vehicles
ADDED - pook boats
ADDED - GNT C185 + variants
ADDED - Qantas C130
ADDED - Usec Bell206 + variants
ADDED - KYC Microlight + Ultralights
ADDED - CYBP_Camel + variants
CHANGED - split vehicles in air/land/sea and adjusted traders
CHANGED - map is now NAPF

0.0.0.3
FIXED - spawn bike (Thanks to Brave Sir Robin)

0.0.0.4
ADDED - overpoch weapons to loot spawns

0.0.0.5
ADDED - Brave Sir Robins Custom buildings

0.0.0.6
CHANGED - moved a lot off code from mission into client
CHANGED - merged multichar.hpp into defines.hpp

0.0.0.7
FIXED - can now add ammo to AH6J
FIXED - cant untow within 5 meters of a wall
FIXED - salvage vehicle exploit (you can now only salvage items that are less than 50% damaged)
ADDED - you can salvage engines and fuel tanks
ADDED - 2 mechs for testing! (buy at hero/bandit traders)
ADDED - AlienZ addon (no there wont be aliens its for the weapons!)
ADDED - Hazmat unit (will give protection from radiation zones )
ADDED - Vilas Colinal space marines addon

0.0.0.8
CHANGED - New loadscreen
FIXED - Moved sigboats to correct category for traders
ADDED - 115+ new skins
ADDED - 8 new vehicles (Including a fire truck!)

0.0.0.9
ADDED - Origins Strongholds, houses and garages (thanks to lunchboxcharlies epoch origins)
ADDED - Can break into vaults
ADDED - fuel tanks now have a limited amount of fuel in them
REMOVED - auto refuel
REMOVED - Bell 206 (caused RPT Spam)

0.0.1.0
ADDED - Single currency banking system
ADDED - Evac uses coins
ADDED - Traders uses coins
ADDED - AI have chance to have coins on them
ADDED - Advanced alchemy building uses coins
ADDED - Single currency admin tools
ADDED - Can transfer coins to other players
ADDED - Souls HUD (temp will be merged into new HUD/DEBUG :P )
ADDED - Can craft from storage (no need to take out items)
ADDED - Paint Cars (need to add ne cars to variable array)

MPMissions/DayZ_Epoch_11.Chernarus/scripts/CustomLoot/CfgLoot.hpp
#include "CfgLootSmall.hpp"
class CfgLoot {
	trash[] = {
		{"TrashTinCan",0.5}
		,{"TrashJackDaniels",0.05}
		,{"ItemSodaEmpty",0.25}
		,{"ItemTrashToiletpaper",0.1}
		,{"ItemTrashRazor",0.1},
		,{"ARP_babywipes",0.2},
		,{"ARP_eyedrops",0.2},
		,{"ARP_lipbalm",0.2},
		,{"ARP_bugspray",0.2},
		,{"ARP_condom",0.2},
		,{"ARP_newspaper",0.2},
		,{"ARP_towel",0.2},
		,{"ARP_canopener",0.2},
		,{"ARP_cigarettes",0.2},
		,{"ARP_basketball",0.2}
	};
	civilian[] = {
		{"FoodCanUnlabeled",0.07}
		,{"TrashJackDaniels",0.08}
		,{"ItemSodaEmpty",0.08}
		,{"ItemSodaCoke",0.09}
		,{"ItemSodaPepsi",0.09}
		,{"FoodCanBakedBeans",0.05}
		,{"FoodCanSardines",0.05}
		,{"FoodCanFrankBeans",0.05}
		,{"FoodCanPasta",0.05}
		,{"8Rnd_9x18_Makarov",0.07}
		,{"7Rnd_45ACP_1911",0.05}
		,{"2Rnd_shotgun_74Slug",0.05}
		,{"2Rnd_shotgun_74Pellets",0.05}
		,{"ItemBandage",0.06}
		,{"ItemPainkiller",0.06}
		,{"FoodBioMeat",0.01}
		,{"8Rnd_9x18_MakarovSD",0.01}
		,{"ItemDocument",0.03}
	};
	office2[] = {
		{"FoodCanUnlabeled",0.07}
		,{"TrashJackDaniels",0.08}
		,{"ItemSodaEmpty",0.08}
		,{"ItemSodaCoke",0.09}
		,{"ItemSodaPepsi",0.09}
		,{"FoodCanBakedBeans",0.05}
		,{"FoodCanSardines",0.05}
		,{"FoodCanFrankBeans",0.05}
		,{"FoodCanPasta",0.05}
		,{"8Rnd_9x18_Makarov",0.07}
		,{"7Rnd_45ACP_1911",0.05}
		,{"2Rnd_shotgun_74Slug",0.05}
		,{"2Rnd_shotgun_74Pellets",0.05}
		,{"ItemBandage",0.06}
		,{"ItemPainkiller",0.06}
		,{"FoodBioMeat",0.01}
		,{"8Rnd_9x18_MakarovSD",0.01}
		,{"ItemBriefcaseEmpty",0.01}
		,{"ItemDocument",0.02}
	};
	food[] = {
		{"TrashTinCan",0.09}
		,{"TrashJackDaniels",0.12}
		,{"ItemSodaEmpty",0.12}
		,{"ItemSodaCoke",0.09}
		,{"ItemSodaPepsi",0.11}
		,{"FoodCanBakedBeans",0.07}
		,{"FoodCanSardines",0.07}
		,{"FoodCanFrankBeans",0.07}
		,{"FoodCanPasta",0.07}
		,{"FoodCanUnlabeled",0.1}
		,{"FoodPistachio",0.05}
		,{"FoodNutmix",0.04}
	};
	office[] = {
		{"FoodCanUnlabeled",0.06}
		,{"ItemSodaEmpty",0.06}
		,{"ItemSodaCoke",0.06}
		,{"ItemSodaPepsi",0.04}
		,{"TrashJackDaniels",0.04}
		,{"FoodCanBakedBeans",0.01}
		,{"FoodCanSardines",0.01}
		,{"FoodCanFrankBeans",0.01}
		,{"FoodCanPasta",0.01}
		,{"ItemWaterbottleUnfilled",0.01}
		,{"ItemWaterbottle",0.01}
		,{"ItemBandage",0.07}
		,{"7Rnd_45ACP_1911",0.03}
		,{"5x_22_LR_17_HMR",0.01}
		,{"10x_303",0.04}
		,{"6Rnd_45ACP",0.04}
		,{"2Rnd_shotgun_74Slug",0.05}
		,{"2Rnd_shotgun_74Pellets",0.05}
		,{"8Rnd_9x18_Makarov",0.08}
		,{"15Rnd_W1866_Slug",0.02}
		,{"WoodenArrow",0.04}
		,{"HandRoadFlare",0.07}
		,{"ItemPainkiller",0.02}
		,{"HandChemGreen",0.01}
		,{"HandChemBlue",0.03}
		,{"HandChemRed",0.03}
		,{"ItemHeatPack",0.03}
		,{"ItemLockbox",0.04}
		,{"ItemDocument",0.02}
	};
	generic[] = {
		{"FoodCanUnlabeled",0.06}
		,{"ItemSodaEmpty",0.06}
		,{"ItemSodaCoke",0.06}
		,{"ItemSodaPepsi",0.04}
		,{"TrashJackDaniels",0.04}
		,{"FoodCanBakedBeans",0.01}
		,{"FoodCanSardines",0.01}
		,{"FoodCanFrankBeans",0.01}
		,{"FoodCanPasta",0.01}
		,{"ItemWaterbottleUnfilled",0.01}
		,{"ItemWaterbottle",0.01}
		,{"ItemBandage",0.11}
		,{"7Rnd_45ACP_1911",0.03}
		,{"5x_22_LR_17_HMR",0.01}
		,{"10x_303",0.04}
		,{"6Rnd_45ACP",0.04}
		,{"2Rnd_shotgun_74Slug",0.05}
		,{"2Rnd_shotgun_74Pellets",0.05}
		,{"8Rnd_9x18_Makarov",0.09}
		,{"15Rnd_W1866_Slug",0.02}
		,{"WoodenArrow",0.04}
		,{"HandRoadFlare",0.07}
		,{"ItemPainkiller",0.02}
		,{"HandChemGreen",0.01}
		,{"HandChemBlue",0.03}
		,{"HandChemRed",0.03}
		,{"ItemHeatPack",0.04}
	};
	medical[] = {
		{"ItemBandage",0.4}
		,{"ItemPainkiller",0.15}
		,{"ItemMorphine",0.2}
		,{"ItemEpinephrine",0.1}
		,{"ItemAntibiotic",0.1}
		,{"ItemHeatPack",0.05}
	};
	hospital[] = {
		{"ItemBandage",0.33}
		,{"ItemPainkiller",0.16}
		,{"ItemMorphine",0.13}
		,{"ItemEpinephrine",0.09}
		,{"ItemBloodbag",0.17}
		,{"ItemAntibiotic",0.12}
	};
	military[] = {
		{"FoodCanUnlabeled",0.07}
		,{"ItemSodaEmpty",0.04}
		,{"ItemSodaCoke",0.01}
		,{"ItemSodaPepsi",0.01}
		,{"ItemBandage",0.04}
		,{"ItemAntibiotic",0.04}
		,{"ItemPainkiller",0.04}
		,{"ItemMorphine",0.01}
		,{"30Rnd_556x45_Stanag",0.04}
		,{"20Rnd_762x51_DMR",0.04}
		,{"17Rnd_9x19_glock17",0.05}
		,{"15Rnd_9x19_M9SD",0.01}
		,{"15Rnd_9x19_M9",0.02}
		,{"30Rnd_762x39_AK47",0.04}
		,{"30Rnd_545x39_AK",0.04}
		,{"5Rnd_762x51_M24",0.01}
		,{"5Rnd_86x70_L115A1",0.01}
		,{"8Rnd_B_Beneli_74Slug",0.04}
		,{"1Rnd_HE_M203",0.01}
		,{"FlareWhite_M203",0.01}
		,{"FlareGreen_M203",0.01}
		,{"1Rnd_Smoke_M203",0.01}
		,{"200Rnd_556x45_M249",0.01}
		,{"HandGrenade_west",0.01}
		,{"HandGrenade_east",0.01}
		,{"SmokeShell",0.04}
		,{"SmokeShellRed",0.02}
		,{"SmokeShellGreen",0.02}
		,{"8Rnd_B_Beneli_Pellets",0.04}
		,{"30Rnd_556x45_StanagSD",0.01}
		,{"30Rnd_9x19_MP5",0.02}
		,{"30Rnd_9x19_MP5SD",0.01}
		,{"100Rnd_762x51_M240",0.01}
		,{"HandChemGreen",0.02}
		,{"HandChemBlue",0.02}
		,{"HandChemRed",0.02}
		,{"ItemHeatPack",0.04}
		,{"FoodMRE",0.03}
		,{"8Rnd_9x18_MakarovSD",0.01}
		,{"20Rnd_B_765x17_Ball",0.04}
		,{"30Rnd_9x19_UZI_SD",0.01}
		,{"ItemDocument",0.02}
	};
	militarypilot[] = {
		{"FoodCanUnlabeled",0.06}
		,{"ItemSodaEmpty",0.04}
		,{"ItemSodaCoke",0.01}
		,{"ItemSodaPepsi",0.01}
		,{"ItemBandage",0.04}
		,{"ItemAntibiotic",0.04}
		,{"ItemPainkiller",0.04}
		,{"ItemMorphine",0.01}
		,{"30Rnd_556x45_Stanag",0.04}
		,{"20Rnd_762x51_DMR",0.04}
		,{"17Rnd_9x19_glock17",0.05}
		,{"15Rnd_9x19_M9SD",0.01}
		,{"15Rnd_9x19_M9",0.02}
		,{"30Rnd_762x39_AK47",0.04}
		,{"30Rnd_545x39_AK",0.04}
		,{"5Rnd_762x51_M24",0.01}
		,{"5Rnd_86x70_L115A1",0.01}
		,{"8Rnd_B_Beneli_74Slug",0.04}
		,{"1Rnd_HE_M203",0.01}
		,{"FlareWhite_M203",0.01}
		,{"FlareGreen_M203",0.01}
		,{"1Rnd_Smoke_M203",0.01}
		,{"200Rnd_556x45_M249",0.01}
		,{"HandGrenade_west",0.01}
		,{"HandGrenade_east",0.01}
		,{"SmokeShell",0.04}
		,{"SmokeShellRed",0.02}
		,{"SmokeShellGreen",0.02}
		,{"8Rnd_B_Beneli_Pellets",0.04}
		,{"30Rnd_556x45_StanagSD",0.01}
		,{"30Rnd_9x19_MP5",0.02}
		,{"30Rnd_9x19_MP5SD",0.01}
		,{"100Rnd_762x51_M240",0.01}
		,{"HandChemGreen",0.02}
		,{"HandChemBlue",0.02}
		,{"HandChemRed",0.02}
		,{"ItemHeatPack",0.04}
		,{"FoodMRE",0.03}
		,{"8Rnd_9x18_MakarovSD",0.01}
		,{"20Rnd_B_765x17_Ball",0.04}
		,{"30Rnd_9x19_UZI_SD",0.01}
		,{"ItemHotwireKit",0.03}
	};
	policeman[] = {
		{"ItemBandage",0.3}
		,{"7Rnd_45ACP_1911",0.08}
		,{"6Rnd_45ACP",0.08}
		,{"15Rnd_W1866_Slug",0.14}
		,{"8Rnd_B_Beneli_Pellets",0.16}
		,{"HandRoadFlare",0.07}
		,{"8Rnd_9x18_MakarovSD",0.01}
		,{"ItemComboLock",0.05}
		,{"SmokeShell",0.04}
		,{"FoodMRE",0.04}
		,{"ItemHotwireKit",0.03}
		,{"ItemDocument",0.02}
	};
	hunter[] = {
		{"ItemBandage",0.4}
		,{"5x_22_LR_17_HMR",0.15}
		,{"7Rnd_45ACP_1911",0.03}
		,{"10x_303",0.1}
		,{"ItemWaterbottleUnfilled",0.05}
		,{"WoodenArrow",0.2}
		,{"ItemHeatPack",0.02}
		,{"FoodMRE",0.01}
		,{"FoodNutmix",0.02}
		,{"ItemDocument",0.01}
		,{"ItemMixOil",0.01}
	};
	worker[] = {
		{"FoodCanUnlabeled",0.09}
		,{"TrashJackDaniels",0.09}
		,{"ItemSodaEmpty",0.1}
		,{"ItemSodaCoke",0.1}
		,{"ItemSodaPepsi",0.1}
		,{"FoodCanBakedBeans",0.06}
		,{"FoodCanSardines",0.06}
		,{"FoodCanFrankBeans",0.06}
		,{"FoodCanPasta",0.06}
		,{"ItemBandage",0.1}
		,{"ItemPainkiller",0.06}
		,{"ItemWire",0.01}
		,{"ItemTankTrap",0.03}
		,{"ItemComboLock",0.04}
		,{"ItemSledgeHead",0.02}
		,{"ItemDocument",0.02}
	};
	clothes[] = {
		{"Skin_RU_Policeman_DZ",0.11}
		,{"Skin_Pilot_EP1_DZ",0.1}
		,{"Skin_Functionary1_EP1_DZ",0.1}
		,{"Skin_Priest_DZ",0.1}
		,{"Skin_Rocker1_DZ",0.07}
		,{"Skin_Rocker2_DZ",0.07}
		,{"Skin_Rocker3_DZ",0.08}
		,{"Skin_Rocker4_DZ",0.08}
		,{"Skin_SurvivorW3_DZ",0.09}
		,{"Skin_SurvivorWpink_DZ",0.09}
		,{"Skin_SurvivorWurban_DZ",0.07}
		,{"Skin_INS_Bardak_DZ",0.02}
		,{"Skin_INS_Worker2_DZ",0.02}
	};
	militaryclothes[] = {
		{"Skin_Camo1_DZ",0.09}
		,{"Skin_Sniper1_DZ",0.08}
		,{"Skin_Rocket_DZ",0.05}
		,{"Skin_Soldier1_DZ",0.5}
		,{"Skin_Drake_Light_DZ",0.07}
		,{"Skin_Soldier_TL_PMC_DZ",0.07}
		,{"Skin_Soldier_Sniper_PMC_DZ",0.05}
		,{"Skin_Soldier_Bodyguard_AA12_PMC_DZ",0.07}
		,{"Skin_CZ_Special_Forces_GL_DES_EP1_DZ",0.08}
		,{"Skin_FR_OHara_DZ",0.08}
		,{"Skin_FR_Rodriguez_DZ",0.09}
		,{"Skin_CZ_Soldier_Sniper_EP1_DZ",0.04}
		,{"Skin_Graves_Light_DZ",0.09}
		,{"Skin_INS_Soldier_AR_DZ",0.05}
		,{"Skin_INS_Soldier_CO_DZ",0.04}
	};
	specialclothes[] = {
		{"Skin_Ins_Soldier_GL_DZ",0.05}
		,{"Skin_GUE_Commander_DZ",0.08}
		,{"Skin_Bandit1_DZ",0.08}
		,{"Skin_Bandit2_DZ",0.06}
		,{"Skin_BanditW1_DZ",0.07}
		,{"Skin_BanditW2_DZ",0.07}
		,{"Skin_TK_INS_Soldier_EP1_DZ",0.05}
		,{"Skin_TK_INS_Warlord_EP1_DZ",0.07}
		,{"Skin_SurvivorWcombat_DZ",0.08}
		,{"Skin_SurvivorWdesert_DZ",0.08}
		,{"Skin_GUE_Soldier_MG_DZ",0.05}
		,{"Skin_GUE_Soldier_Sniper_DZ",0.04}
		,{"Skin_GUE_Soldier_Crew_DZ",0.04}
		,{"Skin_GUE_Soldier_CO_DZ",0.04}
		,{"Skin_GUE_Soldier_2_DZ",0.04}
		,{"Skin_TK_Special_Forces_MG_EP1_DZ",0.02}
		,{"Skin_TK_Soldier_Sniper_EP1_DZ",0.02}
		,{"Skin_TK_Commander_EP1_DZ",0.02}
		,{"Skin_RU_Soldier_Crew_DZ",0.02}
		,{"Skin_INS_Lopotev_DZ",0.02}
	};
	tents[] = {
		{"WeaponHolder_ItemTentOld",0.34}
		,{"WeaponHolder_ItemTentDomed",0.33}
		,{"WeaponHolder_ItemTentDomed2",0.33}
	};
	backpacks[] = {
		{"DZ_Assault_Pack_EP1",0.22}
		,{"DZ_Czech_Vest_Puch",0.2}
		,{"DZ_TerminalPack_EP1",0.18}
		,{"DZ_ALICE_Pack_EP1",0.16}
		,{"DZ_TK_Assault_Pack_EP1",0.13}
		,{"DZ_CompactPack_EP1",0.11}
	};
	militarybackpacks[] = {
		{"DZ_British_ACU",0.3}
		,{"DZ_GunBag_EP1",0.25}
		,{"DZ_CivilBackpack_EP1",0.2}
		,{"DZ_Backpack_EP1",0.15}
		,{"DZ_LargeGunBag_EP1",0.1}
	};
	militaryammo[] = {
		{"2000Rnd_762x51_M134",0.1}
		,{"29Rnd_30mm_AGS30",0.2}
		,{"50Rnd_127x107_DSHKM",0.2}
		,{"48Rnd_40mm_MK19",0.2}
		,{"100Rnd_127x99_M2",0.3}
	};
	pistols[] = {
		{"Makarov",0.25}
		,{"Colt1911",0.23}
		,{"revolver_EP1",0.2}
		,{"glock17_EP1",0.1}
		,{"MakarovSD",0.1}
		,{"M9",0.07}
		,{"M9SD",0.04}
		,{"revolver_gold_EP1",0.01}
	};
	shotgunsingleshot[] = {
		{"Winchester1866",0.25}
		,{"LeeEnfield",0.2}
		,{"MR43",0.2}
		,{"Saiga12K",0.13}
		,{"Remington870_lamp",0.1}
		,{"Crossbow_DZ",0.07}
		,{"M1014",0.05}
	};
	farmweapons[] = {
		{"Winchester1866",0.25}
		,{"LeeEnfield",0.2}
		,{"MR43",0.19}
		,{"Saiga12K",0.13}
		,{"Remington870_lamp",0.1}
		,{"Crossbow_DZ",0.07}
		,{"huntingrifle",0.05}
		,{"Chainsaw",0.01}
	};
	sniperrifles[] = {
		{"AK_107_PSO",0.1}
		,{"M4SPR",0.09}
		,{"M40A3",0.09}
		,{"SVD_CAMO",0.07}
		,{"SVD_des_EP1",0.07}
		,{"SVD",0.07}
		,{"huntingrifle",0.09}
		,{"M24_des_EP1",0.08}
		,{"M24",0.07}
		,{"BAF_LRR_scoped",0.07}
		,{"M107_DZ",0.05}
		,{"KSVK_DZE",0.05}
		,{"VSS_vintorez",0.04}
		,{"SCAR_H_LNG_Sniper_SD",0.03}
		,{"M14_EP1",0.02}
		,{"DMR",0.01}
	};
	sniperriflesammo[] = {
		{"30Rnd_545x39_AKSD",0.18}
		,{"5Rnd_127x108_KSVK",0.08}
		,{"5Rnd_86x70_L115A1",0.07}
		,{"10Rnd_127x99_m107",0.15}
		,{"10Rnd_762x54_SVD",0.2}
		,{"20Rnd_762x51_DMR",0.1}
		,{"5Rnd_762x51_M24",0.15}
		,{"5x_22_LR_17_HMR",0.1}
		,{"20Rnd_762x51_SB_SCAR",0.07}
		,{"20Rnd_9x39_SP5_VSS",0.05}
	};
	submachinegun[] = {
		{"UZI_EP1",0.25}
		,{"bizon",0.2}
		,{"Sa61_EP1",0.2}
		,{"MP5A5",0.13}
		,{"bizon_silenced",0.1}
		,{"UZI_SD_EP1",0.07}
		,{"MP5SD",0.05}
	};
	assaultrifles[] = {
		{"AKS_74_kobra",0.05}
		,{"AKS_74_U",0.05}
		,{"AKS_GOLD",0.01}
		,{"AK_47_M",0.05}
		,{"AK_74",0.05}
		,{"FN_FAL",0.03}
		,{"FN_FAL_ANPVS4",0.01}
		,{"G36A_camo",0.05}
		,{"G36C",0.05}
		,{"G36C_camo",0.05}
		,{"G36K_camo",0.05}
		,{"G36_C_SD_camo",0.03}
		,{"M16A2",0.05}
		,{"M16A2GL",0.02}
		,{"M16A4_ACG",0.01}
		,{"M4A1",0.05}
		,{"M4A1_Aim",0.04}
		,{"M4A1_AIM_SD_camo",0.01}
		,{"M4A1_HWS_GL_camo",0.01}
		,{"M4A3_CCO_EP1",0.04}
		,{"Sa58P_EP1",0.02}
		,{"Sa58V_CCO_EP1",0.04}
		,{"Sa58V_EP1",0.02}
		,{"Sa58V_RCO_EP1",0.04}
		,{"m8_compact",0.03}
		,{"m8_sharpshooter",0.04}
		,{"m8_holo_sd",0.02}
		,{"m8_carbine",0.05}
		,{"BAF_L85A2_RIS_SUSAT",0.02}
		,{"BAF_L85A2_RIS_Holo",0.01}
		,{"M4A1_HWS_GL_SD_Camo",0.01}
	};
	assaultrifleammo[] = {
		{"30Rnd_762x39_SA58",0.25}
		,{"30Rnd_762x39_AK47",0.2}
		,{"30Rnd_556x45_Stanag",0.2}
		,{"30Rnd_545x39_AK",0.11}
		,{"30Rnd_556x45_StanagSD",0.12}
		,{"20Rnd_762x51_FNFAL",0.07}
	};
	machineguns[] = {
		{"RPK_74",0.11}
		,{"M8_SAW",0.11}
		,{"Pecheneg_DZ",0.1}
		,{"PK_DZ",0.1}
		,{"MG36_camo",0.1}
		,{"M60A4_EP1_DZE",0.1}
		,{"MG36",0.1}
		,{"M249_m145_EP1_DZE",0.1}
		,{"m240_scoped_EP1_DZE",0.1}
		,{"M240_DZ",0.08}
		,{"M249_EP1_DZ",0.05}
		,{"Mk_48_DZ",0.05}
	};
	machinegunammo[] = {
		{"100Rnd_556x45_BetaCMag",0.2}
		,{"75Rnd_545x39_RPK",0.2}
		,{"100Rnd_556x45_M249",0.15}
		,{"100Rnd_762x51_M240",0.15}
		,{"100Rnd_762x54_PK",0.15}
		,{"200Rnd_556x45_M249",0.15}
	};
	machinegunammoexpl[] = {
		{"100Rnd_556x45_BetaCMag",0.2}
		,{"75Rnd_545x39_RPK",0.2}
		,{"100Rnd_556x45_M249",0.15}
		,{"100Rnd_762x51_M240",0.15}
		,{"100Rnd_762x54_PK",0.14}
		,{"200Rnd_556x45_M249",0.14}
		,{"MAAWS_HEAT",0.01}
		,{"PipeBomb",0.01}
	};
	militaryshotguns[] = {
		{"Saiga12K",0.25}
		,{"Remington870_lamp",0.35}
		,{"M1014",0.4}
	};
};
